src/bridging/types.ts

```diff
@@ -0,0 +1,177 @@
+import { latest as latestAppData } from '@cowprotocol/app-data'
+import { ethers } from 'ethers'
+import { ChainInfo, ChainId } from '../common'
+import { TokenInfo } from '../common/types/tokens'
+import { Address, OrderKind } from '../order-book'
+import { EvmCall } from '../common/types/ethereum'
+
+export interface BridgeProviderInfo {
+  name: string
+  logoUrl: string
+}
+
+export interface WithSellToken {
+  sellTokenChainId: ChainId
+  sellTokenAddress: Address
+  sellTokenDecimals: number
+}
+
+export interface WithBuyToken {
+  buyTokenChainId: ChainId
+  buyTokenAddress: Address
+  buyTokenDecimals: number
+}
+
+export interface GetBuyTokensParams extends Partial<WithSellToken> {
+  targetChainId: ChainId
+}
+
+/**
+ * Parameters for getting a bridge quote
+ */
+export interface QuoteBridgeRequest extends WithSellToken, WithBuyToken {
+  type: OrderKind.SELL // We make it explicit that only SELL is supported for now
+  amount: string
+  recipient: string
+}
+
+export interface QuoteFee {
+  type: 'fixed' | 'percent'
```

Comment:
I think having only 'percent' is enough, with it and buyAmount we can calculate the fixed value

Reply:
Fair enough. happy to go from simpler to more complex base on need